src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    boolean update(Movie movie);
+
+    boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` or throwing an exception if the deletion fails. Returning a boolean might not be the best approach for error handling.